dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletCacheImpl.java

```diff
@@ -133,6 +135,20 @@ public void remove(String key){
     	
     	String myKey = primaryGroup + key;
     	String metadataKey = metadataGroup + key;
+    	com.dotmarketing.portlets.contentlet.model.Contentlet content = null;
+    	try {
+    		content = (com.dotmarketing.portlets.contentlet.model.Contentlet)cache.get(key,primaryGroup);
+    		try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)